src/Client.ts

```diff
@@ -455,66 +500,88 @@ export default class Client {
   listMessagesPaginated(
     contentTopics: string[],
     opts?: ListMessagesPaginatedOptions
-  ): AsyncGenerator<Message[]> {
+  ): AsyncGenerator<MessageV1[]> {
     const topicFilter = filterForTopics(contentTopics)
-    return mapPaginatedStream(
```

Comment:
extracting listing and paginated listing of envelopes, so that it can be reused for V2.